aws/deployment-role-policy.json

```diff
@@ -31,7 +31,8 @@
         "logs:DescribeLogGroups",
         "logs:DeleteLogGroup",
         "logs:PutRetentionPolicy",
-        "logs:CreateLogGroup"
+        "logs:CreateLogGroup",
+        "logs:TagResource"
```

Comment:
How about `logs:UntagResource`?

Reply:
`logs:UntagResource` we not used in policy.json([url](https://dsgithub.trendmicro.com/file-storage-security/cloudformation-templates/pull/166/files)), i think that is not required .